### DISCUSSION TITLE
Suppression des tarifs Total Standard Fixe ?

### DISCUSSION THREAD

**Discussion by zdimension** [JC144/EDF_Simulateur_Prix#87] [General]
Bonjour,
J'ai vu que les tarifs Standard Fixe de Total ont été retirés dans le commit https://github.com/JC144/EDF_Simulateur_Prix/commit/a6ca076d755692cbe32275c4fec747b829d31729, je me demandais pourquoi ?

**Comment by JC144** [comment]
Lors de la mise à jour, je n'ai pas trouvé la fiche tarifaire de cette offre.
Je viens de la voir, j'essaierai de la rajouter quand j'aurai un peu de temps ou tu peux faire une PR.

Je pense qu'à terme, je vais séparer les tarifs EDF des tarifs non-EDF et rajouter une case à cocher pour simuler les tarifs communautaires.
Cela devient lourd à maintenir et les opérateurs ne veulent pas jouer le jeu.
